sources/packages/backend/apps/api/src/main.ts

```diff
@@ -69,7 +69,6 @@ async function bootstrap() {
         },
         "access-token",
       )
-      .addServer("/api")
```

Comment:
Adding screenshot to explain if anyone having doubts on this. 

![image](https://user-images.githubusercontent.com/54600590/217619063-3490eb7e-d6e0-4518-bf7d-916444d583d5.png)

Reply:
Thanks @dheepak-aot, please see below also the double `api` in the URL that was causing all calls to fail.
![image](https://user-images.githubusercontent.com/61259237/217619682-75e0cdd1-2541-410f-b043-4b9858a9966b.png)